### PR TITLE
docs(split): expands design guidelines for split layout.

### LINKED ISSUE #3587: Enhance design guidelines for our Layouts pages - Split
Add guidelines to Split Layouts page
Include picture example of Split being used

### PR DESCRIPTION
**Pull request by edonehoo**
closes #3587

### CODE REVIEW

File: packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/split.md [hunk 1]
```diff
@@ -2,4 +2,32 @@
...
+1. **Split item**: An individual section of content to be placed in the split layout.
+
+## Usage
+Use the split layout when content should be flexible and may change dynamically. 
+
+## Variations
+
+### Multiple items that fill space 
+
+You can specify which split item fills the remaining container space. More than one item can be set to fill, and they will share the remaining space equally.
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/split.md:21]:
```suggestion
You can specify which split item fills the remaining container space. More than one item can be set to fill and they will share the remaining space equally.
```

File: packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/split.md [hunk 2]
```diff
@@ -2,4 +2,32 @@
...
+
+<img src="./img/split-filled.png" alt="PatternFly website screenshot showing modal created with bullseye layout." width="3330px"/>
+
+### Items that wrap 
+
+You can wrap a split item to ensure that it fits among other content in the viewport as it is resized.
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/split.md:27]:
```suggestion
You can wrap a split item to ensure that it properly fits alongside other content as the screen is resized.
```

**Comment by edonehoo** [comment]
@lboehling @mmenestr initial drafts of the layouts design guidelines are ready if you two want to take a look & give feedback! I'll tag you on all of them just fyi when you get a lot of notifs. I couldn't find images for most layouts (and will need to confirm the ones I found with devs), so if you have ideas on where to grab some I'm all ears

File: packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/split.md [hunk 3]
```diff
@@ -2,4 +2,30 @@
...
+1. **Split item**: An individual section of content to be placed in the split layout.
+
+## Usage
+Use the split layout when content should be flexible and may change dynamically. 
+
+## Variations
+
+### Multiple items that fill space 
+
+You can specify which split item fills the remaining horizontal space in the layout. More than one item can be set to fill, as, and they will share the remaining space equally.
```
Comment (mmenestr) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/split.md:21]: I don't understand this sentence haha the "can be set to fill, as, and"

**Review by mmenestr — CHANGES_REQUESTED** [review]
just have 1 comment to review, otherwise looks good!

File: packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/split.md (hunk 3 above)
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/split.md:21]:
```suggestion
You can specify which split item fills the remaining horizontal space in the layout. More than one item can be set to fill, and they will share the remaining space equally.
```

**Review by mmenestr — APPROVED** [review]

**Pull request merged**